lra/lrademo/trip-manager-springboot/src/main/resources/application.properties

```diff
@@ -26,3 +26,9 @@ spring.microtx.lra.headers-propagation-prefix = {x-b3-, oracle-tmm-, authorizati
 
 hotel.service.url = http://localhost:8082/hotelService/api/hotel
 flight.service.url = http://localhost:8083/flightService/api/flight
+
+hotel.service.name = hotel-service
+flight.service.name = flight-service
+
```

Comment:
Add a comment here saying that the interceptor can also be configured using configuration.

Reply:
Done